docs/HowToUseSmartthingsSDK.md

```diff
@@ -0,0 +1,23 @@
+How to use SmartThings SDK
+===================
+
+Lightweight OCF development framework based Tizen RT.
+
+----------
+
+
+Introduction
+-------
+SmartThings SDK supports Samsung Connect Services - provided by Samsung mobile devices. Through the SmartThings Console, you can easily connect your device to the cloud. It also provides an environment to test the developed automation app.
+
+How to use
+------
+####1. Using Sample App
+Please find it [README](../apps/examples/st_things/README.md) to build st_things sample app.
+
+####2. Using Tizen Studio 2.0 for RT IDE
```

Comment:
1. How about below?
```
2. Using Tizen Studio for RT IDE
blah blah
Tizen Studio supports SmartThings SDK since v2.0.
```
2. Add two spaces.
3. Option (tab) level of Howtouse is wrong.
We can use a sample app on RT IDE.